packages/cna-template/template/nuxt/nuxt.config.js

```diff
@@ -68,6 +73,11 @@ export default {
     <%_ } _%>
   ],
   /*
+  ** Auto import components
+  ** See https://nuxtjs.org/api/configuration-components
+  */
+  components: true,
```

Comment:
Is Nuxt.js officially suggesting using auto-detected components now ?

Can this pass eslint no-undef rule ?

Reply:
I like to see it as an option, could also be promptable in the create-nuxt-app script

Reply:
Actually it's really easy to disable, just set it to `false` ;)